python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py

```diff
@@ -17,10 +17,13 @@
 import logging
 from . import driver
 import re
+import exceptions
+
+# Globals
+apilevel = '2.0'  # String constant stating the supported DB API level.
```

Comment:
Remove the comment which is duplicated with pep document.

Reply:
updated